conbench/templates/index.html

```diff
@@ -119,8 +119,8 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
           "searchPlaceholder": "commit hash works too",
       }
 });
-
 enable_search_query_string('#runs');
+specific_column_search('#runs', [3,6]);
```

Comment:
Could you add a code comment for what this does? Something like: "in the Runs table, add a setpicker to the headers of columns three and six, which are ..." (I don't know if I have understood this!)